drivers/hud_apr/spec/models/fy2024/datalab_2_0_spec.rb

```diff
@@ -43,7 +43,7 @@ def run(generator, filter)
   if File.exist?('drivers/datalab_testkit/spec/fixtures/inputs/merged/source/Export.csv')
     include_context 'datalab multiple projects apr'
     include_context 'datalab organization a rrh apr'
-    # include_context 'datalab organization o sso apr'
+    include_context 'datalab organization o sso apr'
     # include_context 'datalab organization o th apr'
```

Comment:
If the goal is to avoid accidentally missed tests, maybe we should push this down into `xit`s or something instead of commenting them out? Not ideal, but at least we get a message...

Reply:
Yes, but these haven't been QA'd at all yet.  Once we've completed the first pass, we should uncomment them.